### PR TITLE
R-snowfall: new package

### PR DESCRIPTION
**Pull request by baberlevi**
required dependency for phantompeakqualtools
somehow #6403 made it through without it

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-snowfall/package.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RSnowfall(RPackage):
+    """Usability wrapper around snow for easier development of parallel R
```
Comment (healther) [inline var/spack/repos/builtin/packages/r-snowfall/package.py]: Can you separate the first line out? Spack uses the first line for the short information and the rest of the docstring for the info message

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/r-snowfall/package.py]: Where do we do this? No other package in Spack does this.

Comment (baberlevi) [reply var/spack/repos/builtin/packages/r-snowfall/package.py]: so - don't change it?

**Comment by baberlevi** [comment]
bumping for a merge please

**Pull request merged**